packages/crawler/package.json

```diff
@@ -0,0 +1,17 @@
+{
+  "name": "crawler",
+  "version": "1.0.0",
+  "description": "Porsche Design System Crawler",
+  "repository": "git@github.com:porsche-design-system/crawler.git",
+  "author": "Dr. Ing. h.c. F. Porsche AG",
+  "license": "MIT",
```

Comment:
- look in another package: license, private, version 0.0.0, repository
- remove prettier
- check devendencies, puppeteer is missing, shared package

Reply:
Done